tests/integration/conftest.py

```diff
@@ -391,6 +391,23 @@ def vmc_nsx_connect(vmc_config):
     )
 
 
+@pytest.fixture(scope="session")
+def vmc_vcenter_connect(vmc_config):
```

Comment:
updated it to just return `vmc_config["vmc_vcenter_connect"]`

This fixture will be used in tests for other modules which will be part of future PRs

Reply:
FWIW it's a bad idea to add test code that isn't used. Just like it's a bad idea to add regular code that isn't used or tested.

Code should only be added in the PR that needs that code, not before. This time it's not a blocker, but next time it will be.